.github/workflows/build.yml

```diff
@@ -65,17 +65,19 @@ jobs:
       #  run: gpg --quiet --batch --yes --decrypt --passphrase="$GPG_SECRET_PASSPHRASE" --output spotbugs.jks .github/workflows/spotbugs.jks.gpg;
       #  env:
       #    GPG_SECRET_PASSPHRASE: ${{ secrets.GPG_SECRET_PASSPHRASE }}
+      - name: Pre-Build verification
```

Comment:
no on all this.

Reply:
agreed upon PMD applying the same best practice ignoring the obsolete (impl) detail as going to change anytime soon. If PMD or rewrite fails the build does not matter. We only talking about dryRun which makes no harm at all. 

No one forces to apply the run goal which of course should be the natural outcome, but still is in the responsibility of each dev to check each commit.